recipe/meta.yaml

```diff
@@ -92,7 +92,6 @@ extra:
   recipe-maintainers:
     - teo-tsirpanis
     - jdblischak
-    - KiterLuc
     - akrherz
```

Comment:
Does anyone have context on this line BTW?

Reply:
Sorry -- I meant -- does anyone have context on `akrherz`?

Reply:
(That's a name I do not recognize)

Reply:
conda-forge maintainer